tech/languages/ruby/gems-installation.md

```diff
@@ -73,3 +73,19 @@ To look whether the required gem is available as an RPM package, look at
 [IsItFedoraRuby.com](http://isitfedoraruby.com/) site which tracks the Ruby
 integration within the Fedora project by listing available information about
 packaged gems.
+
+## Installing gems as a regular user
+
+You might want to install gems on your user's directory; as opposed to the system's, by default. If so, edit `$HOME/.gemrc` like this:
+
+```
+gem: --user-install
```

Comment:
Hmm, after reading up on this a bit,
there is another notation that does not override the globals:
```
install: --user-install
```
This preserves what the [operating_system.rb](https://src.fedoraproject.org/rpms/ruby/blob/rawhide/f/operating_system.rb#_103) sets.

with
```
$ echo "install: --no-documents" > ~/.gemrc
```
`gem env` then reports following:
```
     - "gem" => "--user-install --bindir /home/jprokop/bin" # <= default set by the mentioned `operating_system.rb`
     - "install" => "--no-document" # <= We set this previously in `.gemrc`
```

Reply:
I'll leave it up to you whether or not to use it and leave the system's defaults intact (if left intact the `--bindir` that is set also stays and bins get generated to `$HOME/bin`)

Reply:
This is interesting. I didn't know this. Maybe it would be wise to mention all so that if the user creates `~/.gemrc` knows about said defaults. 

Thank you for all the corrections, btw.

Reply:
OK, it seems that my recommendation is useless if we use `install: --no-document`. I'll just fix that recommendation instead. 

Would it be worth leaving this as a notice, maybe? Just to let users know what happens when you use: `gem: --no-document` or such notation?

Reply:
+1 for preserving the system defaults and using `install:` instead.

Reply:
> OK, it seems that my recommendation is useless if we use `install: --no-document`. I'll just fix that recommendation instead.
> 
> Would it be worth leaving this as a notice, maybe? Just to let users know what happens when you use: `gem: --no-document` or such notation?

+1 for using `install: ...`. I like mentioning that `gem: ...` will override some defaults that Fedora has.